### PR TITLE
Issue #576 fix - Servermetrics module

### PR DESCRIPTION
**Pull request by tuxudo**
Fix for Servermetrics module on 10.11+ Macs.

### CODE REVIEW

File: app/modules/servermetrics/scripts/servermetrics.py [hunk 1]
```diff
@@ -66,6 +66,10 @@ def __main__():
     if getOsVersion() < 10:
         logFormat = 0
 
+    if getOsVersion() < 11:
```
Comment (tuxudo) [inline app/modules/servermetrics/scripts/servermetrics.py]: Fixed

File: app/modules/servermetrics/scripts/servermetrics.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
```
Comment (tuxudo) [inline app/modules/servermetrics/scripts/servermetrics.sh]: Added null handling.

File: app/modules/servermetrics/scripts/servermetrics.sh [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+# Crude Code™ by @tuxudo
```
Comment (tuxudo) [inline app/modules/servermetrics/scripts/servermetrics.sh]: I don't know python :/

**Comment by bochoven** [comment]
Thanks!

**Pull request merged**